hwdb/60-evdev.hwdb

```diff
@@ -130,6 +132,15 @@ evdev:name:Elan Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnUX305UA:*
  EVDEV_ABS_36=0:2119:33
 
 #########################################
+# Chalkboard Electronics
+#########################################
+evdev:name:Chalkboard Electronics HID Touchscreen*:dmi:*
+ ID_INPUT_TABLET=0
+ ID_INPUT_TOUCHSCREEN=1
+ EVDEV_ABS_00=::6
+ EVDEV_ABS_01=::6
+
```

Comment:
does this work? 60-evdev.rules is run before 60-input-id.rules, so I'd expect the latter to overwrite whatever you set here. Honestly, I think we're better off adding a few rules to 60-input-id and having a 60-input-id-overrides.hwdb. Mixing this with the evdev bits is likely to hurt us long-term